arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2016-02-03",
+		"title": "iotHubClient",
+		"description": "API for using the IoT Hub Features"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"paths": {
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}": {
+			"get": {
+				"tags": ["GET"],
+				"summary": "Get an IotHub",
+				"description": "Get an IotHub",
+				"operationId": "IotHubResource_GetIotHub",
+				"consumes": [],
+				"produces": ["application/json", "text/json"],
+				"parameters": [{
+					"name": "resourceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "subscriptionId",
```

Comment:
subcriptionId should be a global parameter as it must be a property on the client. 
Take a look at this example: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-storage/2016-01-01/swagger/storage.json#L978